frontend/src/components/Block/Block.jsx

```diff
@@ -161,11 +161,11 @@ const Experiment = ({ match }) => {
         state,
     });
 
-    // Render experiment state
+    // Render block state
     const render = (view) => {
         // Default attributes for every view
         const attrs = {
-            experiment,
+            experiment: block,
```

Comment:
I know this is a hassle, but could the subcomponents not also already be updated to use `block` instead of `experiment`?